Modules/mmapmodule.c

```diff
@@ -729,7 +729,7 @@ mmap_seek_method(mmap_object *self, PyObject *args)
         if (where > self->size || where < 0)
             goto onoutofrange;
         self->pos = where;
-        Py_RETURN_NONE;
+        return PyLong_FromLong(self->pos);
```

Comment:
```suggestion
        return PyLong_FromSsize_t(self->pos);
```

See the warning: 'function': conversion from 'Py_ssize_t' to 'long', possible loss of data

Reply:
Already did :0

Reply:
See: https://github.com/python/cpython/pull/111852/commits/55f47cb51d41a385b5ca5199da3890e631c5a553